newrelic/resource_newrelic_alert_policy_channel.go

```diff
@@ -116,9 +118,26 @@ func resourceNewRelicAlertPolicyChannelCreate(ctx context.Context, d *schema.Res
 		return diag.FromErr(err)
 	}
 
+	retryErr := resource.RetryContext(ctx, d.Timeout(schema.TimeoutCreate), func() *resource.RetryError {
```

Comment:
Should we define a value for schema.TimeoutCreate? I wasn't sure if there was a default value.

Reply:
I think it has a default, but I added it to be sure. Thanks 👍